src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -5868,7 +5869,7 @@ export interface NS {
    * RAM cost: 0.1 GB
    *
    * Returns a boolean indicating whether the specified file exists on the target server.
-   * The filename for programs is case insensitive, other file types are case sensitive.
+   * The filename for programs is case-insensitive, other file types are case-sensitive.
```

Comment:
predicate adjectives (describes what the subject "is") aren't hyphenated. In general there is no ambiguity in the meaning, so no reason to hyphenate "case-sensitive" in other cases either, but those are more style choices.